libs/template/src/lib/core/utils/template-manager_creator.ts

```diff
@@ -86,6 +95,14 @@ export function createTemplateManager<C extends object, N extends string = strin
       }
     },
 
+    getEmbeddedView(name: N): EmbeddedViewRef<C> {
+      if (viewCache.has(name)) {
+        return viewCache.get(name);
+      } else {
+        return undefined;
```

Comment:
Nitpick: `Map.prototype.get()` seems to return `undefined` by itself when no value is provided for a given key.

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/get